src/Adapter/Console.php

```diff
@@ -164,16 +164,24 @@ public function __destruct()
     public function setOutputStream($resource)
     {
         $fileOpenError = null;
-        ErrorHandler::start();
-        try {
-            $stream = fopen($resource, 'w');
-        } catch (ValueError $fileOpenError) {
+        if (is_null($resource)) {
             $stream = false;
+        } else {
+            ErrorHandler::start();
+            try {
+                $stream = fopen($resource, 'w');
+            } catch (ValueError $fileOpenError) {
+                $stream = false;
```

Comment:
Perhaps pre-assign `$stream = false`? :thinking: 

We can use a `} finally {` block to restore the error handler

Reply:
Nice suggestion